api/buckets/pb/buckets.proto

```diff
@@ -195,27 +191,27 @@ message ArchiveWatchRequest {
     string key = 1;
 }
 
-message ArchiveWatchReply {
+message ArchiveWatchResponse {
     string msg = 1;
 }
 
-service API {
-    rpc List(ListRequest) returns (ListReply) {}
-    rpc Init(InitRequest) returns (InitReply) {}
```

Comment:
`Init` is no good in Obj C and Swift because its a reserved name. Went with `Create`, but let me know if there is something better.